### PR TITLE
Clarify `carryoverConcurrencyCount` documentation

### PR DESCRIPTION
**Pull request by Rafael09ED**
Updates `carryoverConcurrencyCount` documentation as discussed in issue #99 .

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ The length of time in milliseconds before the interval count resets. Must be fin
 Type: `boolean`\
 Default: `false`
 
-Whether the task must finish in the given interval or will be carried over into the next interval count.
+If `true`, specifies that an incomplete `Promise` should be carried over into the next interval and counted against the `intervalCap`. If `false`, an incomplete promise will continue to run independent of the next interval and will not count towards the `intervalCap`.
```
Comment (szmarczak) [inline readme.md]: What do you mean by `incomplete`?

Comment (Rafael09ED) [reply readme.md:108]: A promise that has started executing but hasn't settled yet.

Comment (sindresorhus) [reply readme.md:108]: @Rafael09ED Can you clarify in the text what you mean by "incomplete" or even better, use a better term than "Incomplete".

**Comment by sindresorhus** [comment]
@Rafael09ED Bump

File: readme.md (hunk 1 above)
Comment (Rafael09ED) [reply readme.md:108]: How's:
If `true`, specifies that any [pending](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) Promises, should be carried over into the next interval and counted against the `intervalCap`. If `false`, any of those pending Promises will not count towards the next `intervalCap`.

Comment (sindresorhus) [reply readme.md:108]: 👍

**Comment by Rafael09ED** [comment]
Do I need to merge `sindresorhus:master` into `Rafael09ED:patch-1` or would the pull request be fine since the changes are localized to one line?

**Pull request merged**